src/components/AddPaymentCard/PaymentCardForm.tsx

```diff
@@ -286,7 +282,6 @@ function PaymentCardForm({
                     label={translate('common.zip')}
                     aria-label={translate('common.zip')}
                     role={CONST.ROLE.PRESENTATION}
-                    inputMode={CONST.INPUT_MODE.NUMERIC}
```

Comment:
I removed this line because on mWeb platforms this would only allow numeric input which would restrict users from typing UK Zip Codes what contain characters and numbers (ex. SW8 4LT). The issue was present on iOS: mWeb as well, here's an example from Android: mWeb before and after removing the line:

| Before | After |
| ------------- | ------------- |
| <img width="375" alt="android-mweb-before" src="https://github.com/user-attachments/assets/4a62c1bd-7a2c-4aa4-877a-db6a7e911ac7"> | <img width="375" alt="android-mweb-after" src="https://github.com/user-attachments/assets/ab4b4307-acb9-47fd-84d4-4d86c2b01c12"> |

Reply:
that makes sense, thanks